client/src/components/inputs/Button.js

```diff
@@ -1,34 +1,31 @@
-import React, { Component } from 'react';
-import PropTypes from 'prop-types';
+import React, { Component } from "react";
+import PropTypes from "prop-types";
 
 const propTypes = {
-    onClick: PropTypes.func,
-    size: PropTypes.oneOf(['small', 'medium', 'large']),
-    kind: PropTypes.oneOf(['primary', 'secondary', 'tertiary']),
-    content: PropTypes.string.isRequired,
-    type: PropTypes.string
+  onClick: PropTypes.func,
+  size: PropTypes.oneOf(["small", "medium", "large"]),
+  kind: PropTypes.oneOf(["primary", "secondary", "tertiary"]),
+  content: PropTypes.string.isRequired,
+  type: PropTypes.string
 };
 
 const defaultProps = {
-    size: 'medium',
-    kind: 'primary'
+  size: "medium",
+  kind: "primary"
 };
 
 class Button extends Component {
-    // constructor(props) {
-    //     super(props);
-    // }
-
-    render() {
-        return (
-            <button
-                onClick={this.props.onClick}
-                className={`button ${this.props.size} ${this.props.kind}`}
-                type={this.props.type} >
-                {this.props.content}
-            </button>
-        );
-    }
+  render() {
```

Comment:
is it defined somewhere that our tabs are two spaces?

Reply:
Yeah that is the default prettier config, but in the #100 pr, I changed it to 4 because that is equivalent to a tab, and I think generally, it is easier to read this way.